docs/prover/cli/options.md

```diff
@@ -688,6 +688,54 @@ struct TokenPair {
 We have two contracts `BankToken.sol` and `LoanToken.sol`. We want `tokenA` of the `tokenPair` to be `BankToken`, and `tokenB` to be `LoanToken`. Addresses take up only one slot. We assume `tokenPair` is the first field of Bank (so it starts at slot zero). To do that, we use:
 `certoraRun Bank.sol BankToken.sol LoanToken.sol --verify Bank:Bank.spec --structLink Bank:0=BankToken Bank:1=LoanToken`
 
+(--contract_recursion_limit)=
+### `--contract_recursion_limit`
+
+**What does it do?**
+Contract inlining can cause recursion (see {ref}`--optimistic_contract_recursion`). This
+option sets the contract recursion level, which is the number of recursive calls
+that the Prover will consider when inlining contracts linked using e.g. `--link` or `--struct_link`.
```

Comment:
```suggestion
that the Prover will consider when inlining contracts linked using, e.g., `--link` or `--struct_link`.
```

Reply:
What if the recursion happens without linking, for example if a function called in main contract (`currentContract`) is already recursive?  Maybe that's already considered linking, but it's not very clear.

Reply:
In what sense recursive? Internal calls? I can clarify that's not the kind of recursion this option deals with, it deals with recursion from external calls.